src/js/components/landing/recentActivity/DeleteModal.jsx

```diff
@@ -0,0 +1,102 @@
+/**
+  * DeleteModal.jsx
+  * Created by Minahm Kim 03/10/17
+  **/
+
+import React from 'react';
+import Modal from 'react-aria-modal';
+import { hashHistory } from 'react-router';
+import * as Icons from '../../SharedComponents/icons/Icons.jsx';
+
+import * as ReviewHelper from '../../../helpers/reviewHelper.js';
+
+export default class DeleteModal extends React.Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			closeable: true,
```

Comment:
Is this and all checks related to it even relevant/necessary? It never changes, just stays "true" always as far as I can tell.